core/execution/engine.go

```diff
@@ -1137,6 +1290,16 @@ func (e *Engine) OnTick(ctx context.Context, t time.Time) {
 		}
 		evts = append(evts, events.NewMarketDataEvent(ctx, mkt.GetMarketData()))
 	}
+
+	for _, mkt := range e.spotMarketsCpy {
```

Comment:
I guess theres not any successor market's stuff in this PR, there is a very brief mention of it in the successor market spec. We should open a ticket for it so its not forgotten.

Reply:
successor markets are not going to be supported for spots at this point. It was agreed to be pushed back.